l2geth/rollup/sync_service.go

```diff
@@ -676,7 +676,7 @@ func (s *SyncService) applyTransactionToTip(tx *types.Transaction) error {
 		bn := tx.L1BlockNumber()
 		s.SetLatestL1Timestamp(ts)
 		s.SetLatestL1BlockNumber(bn.Uint64())
-		log.Debug("Updating OVM context based on new transaction", "timestamp", ts, "blocknumber", bn.Uint64(), "queue-origin", tx.QueueOrigin().Uint64())
+		log.Debug("Updating OVM context based on new transaction", "timestamp", ts, "blocknumber", bn.Uint64(), "queue-origin", tx.QueueOrigin())
```

Comment:
If we implemented `Stringer`, then we could do `tx.QueueOrigin().String()` here and have a nice human readable string

Reply:
So I added the `Stringer` and I'm getting:

```
tx.QueueOrigin().String undefined (type types.QueueOrigin has no field or method String)
```

Should this just automatically be converted to the string based on the `Stringer`?